src/components/TaskCard/index.module.scss

```
@@ -12,14 +12,55 @@
   height: 10.188rem;
   border: 0.063rem solid $lilac-blue;
   border-radius: 0.313rem;
+  position: relative;
 }
+
+.task {
+  font-size: 1.5rem;
+  font-weight: 700;
+  line-height: 1.758rem;
+  color: $navy-blue;
+}
+.taskDone {
```

IMHO, new line after each css property looks more intuitive.
```suggestion
}

.taskDone {
```